historical-bank.gemspec

```diff
@@ -44,7 +44,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'money',    '~> 6.7'
   s.add_runtime_dependency 'httparty', '~> 0.14'
-  s.add_runtime_dependency 'redis',    '~> 3.3'
+  s.add_runtime_dependency 'redis',    '>= 3.3'
```

Comment:
as this might break in future versions of the gem, can you change this to 
```ruby
  s.add_runtime_dependency 'redis',    ['>=3.3', '< 4.1']
```
instead?